app/views/featured_attachments/index.html.erb

```diff
@@ -24,6 +24,18 @@
         <%= t("featured_attachments.index.featured_attachments.description_govspeak") %>
       </p>
 
+      <p class="govuk-body">
+        <%= link_to(
+          "Reorder attachments",
+          reorder_featured_attachments_path(@edition.document),
+          class: "govuk-link",
+          data: { gtm: "reorder-attachments" }
+        ) %>
+      </p>
+
+      <br/>
+      <br/>
```

Comment:
Are these present to create some additional margin? I'd assume there's a better class based approach we can use for this.

Reply:
Yeah, the design calls for it, and it does look worse without them. I'd like to get @alex-ju's opinion on what we can do here. I tried `govuk-!-margin-bottom-8`, but it wasn't enough. I need more margin bottom!

Reply:
Yup, good call. Alex should have some ideas for this.

Reply:
The prototype uses `govuk-body govuk-!-margin-bottom-8` – I don't really like seeing overwrites, but the design calls for it as you mentioned.

Reply:
Ah, OK. Strange, I thought it wasn't working when I tried it - updated, thanks @alex-ju 👍